server/main.py

```diff
@@ -31,14 +33,40 @@ class Payload(BaseModel):
     tvib: Optional[float] = None
     trot: Optional[float] = None
     path_length: float
-    simulate_slit: Optional[int] = None
-    mode: Literal["absorbance", "transmittance_noslit", "radiance_noslit", "transmittance", "radiance"]
+    simulate_slit: int
+    use_simulate_slit:bool
+    mode: Literal["absorbance", "transmittance_noslit", "radiance_noslit"]
```

Comment:
Why change the type for mode here? Don't we still pass "transmittance" and "radiance" as mode?